src/components/DropdownMenu.tsx

```diff
@@ -25,10 +25,7 @@ export const DropdownMenuButton = forwardRef<
   return (
     <button
       className={clsxm(
-        "group flex items-center gap-2 stroke-white hover:stroke-orange-400 hover:text-orange-400",
-        {
-          "stroke-orange-400 text-orange-400": props["aria-expanded"] === true,
-        },
+        "group flex items-center gap-2 stroke-white hover:stroke-orange-400 hover:text-orange-400 ui-open:stroke-orange-400 ui-open:text-orange-400",
```

Comment:
You can see the new classes from the `@headlessui/tailwindcss` plugin here: `ui-open:`. You should be able to find the relevant classes for a component you're working with in the headless docs.